engine/installation/linux/docker-ce/centos.md

```diff
@@ -77,31 +77,38 @@ from the repository.
     ```
 
 2.  Use the following command to set up the **stable** repository. You always
-    need the **stable** repository, even if you want to install **edge** builds
-    as well.
+    need the **stable** repository, even if you want to install builds from the
+    **edge** or **testing** repositories as well.
```

Comment:
With "stable" builds now also being pushed to the "edge" channel, basically;

- "stable" only gives you stable releases
- "edge" gives you both "stable" and "edge" releases
- "test" gives you "stable", "edge" and "release-candidates"

All channels should give you the latest version by default (unless "pinned" to a specific version)

We can tweak after release though